client/src/pages/Login.js

```diff
@@ -0,0 +1,166 @@
+import React from "react";
+import {
+  Container,
+  Typography,
+  Box,
+  TextField,
+  Divider,
+  Button,
+  Checkbox,
+  FormControlLabel,
+} from "@material-ui/core";
+import Grid from "@material-ui/core/Grid";
+import { makeStyles } from "@material-ui/core/styles";
+import { Link } from "react-router-dom";
+
+import NavBar from "../components/Navbar";
+
+import { validateEmail } from "../util/validateEmail";
+
+const useStyles = makeStyles((theme) => ({
+  header: {
+    padding: "2rem",
+    borderBottom: "2px",
+    borderBottomColor: theme.primary,
+  },
+  signupLink: {
+    marginLeft: "2px",
+  },
+  subtitle: {
+    padding: "2rem",
+  },
+  divider: {
+    backgroundColor: theme.primary,
+    height: "2px",
+    width: "10%",
+    margin: "auto",
+  },
+  button: {
+    backgroundColor: theme.primary,
+    marginTop: "4rem",
+    marginBottom: "4rem",
+    color: theme.bgcolor,
+    height: "3rem",
+    width: "60%",
+  },
+}));
+
+function Login(props) {
+  const classes = useStyles();
+
+  const [state, setState] = React.useState({
```

Comment:
I'm guessing you put it all in one object so that when you integrate it in the background, you can just send the whole state? 

if not, i would separate out each key to its own state and create a final object before sending to the backend

Reply:
This did originally seem like a cleaner approach as it used less code, but upon further research it seems that the useState hook replaces the state variable when called rather than merging changes, so replacing the entire object would be inefficient. It makes sense that I separate out each key, I will make this change!